### ISSUE TITLE
RangeError: Maximum call stack size exceeded in proposals-ideas-searchbox.js:84

### ISSUE THREAD

**Issue by cdparra** [socialappslab/appcivist-pb-client#748]
![image](https://user-images.githubusercontent.com/630150/27886049-5e663824-61a8-11e7-9e8b-e2a745e4bfc4.png)

```
RangeError: Maximum call stack size exceeded
    at Scope.searchMode (proposals-ideas-searchbox.js:84)
    at Scope.setMode (proposals-ideas-searchbox.js:262)
    at Scope.searchMode (proposals-ideas-searchbox.js:91)
    at Scope.setMode (proposals-ideas-searchbox.js:262)
    at Scope.searchMode (proposals-ideas-searchbox.js:91)
    at Scope.setMode (proposals-ideas-searchbox.js:262)
    at Scope.searchMode (proposals-ideas-searchbox.js:91)
    at Scope.setMode (proposals-ideas-searchbox.js:262)
    at Scope.searchMode (proposals-ideas-searchbox.js:91)
    at Scope.setMode (proposals-ideas-searchbox.js:262)
```

**Comment by jorgeramirez** [comment]
@cdparra already seen this error. I will fix it when pushing #706

**Comment by cdparra** [comment]
Ok @jorgeramirez moving to `In progress` and clearing the estimate as it is part of another issue
